### DISCUSSION TITLE
How to handle the env variable when CI/CD action?

### DISCUSSION THREAD

**Discussion by ambitious922** [community/community#58316] [Actions] [answered]
### Select Topic Area

Question

### Body

When handle the action in GitHub, how to handle the env variable?

**Comment by fourfor4** [comment, marked as answer]
Good Question, 
If you are admin, you can manage the secret keys in the settings.
And you have to manage the workflow file with that secret keys.

That's all

**Reply by ambitious922** [reply]
Good answer.
Thanks.

**Comment by akashammm** [comment]
Akashvai5
